solr/solr-ref-guide/src/coreadmin-api.adoc

```diff
@@ -50,6 +94,45 @@ The `CREATE` action creates a new core and registers it.
 If a Solr core with the given name already exists, it will continue to handle requests while the new core is initializing. When the new core is ready, it will take new requests and the old core will be unloaded.
 
 `admin/cores?action=CREATE&name=_core-name_&instanceDir=_path/to/dir_&config=solrconfig.xml&dataDir=data`
+[.dynamic-tabs]
+--
+[example.tab-pane#v1coreadmin-create]
+====
+[.tab-label]*V1 API*
+
+Assuming you are using an existing configSet to create your new core:
+[source,bash]
+----
+http://localhost:8983/solr/admin/cores?action=CREATE&&name=techproducts_v2&configSet=sample_techproducts_configs
+
+----
+
+If you have already existing core files deployed on disk, and need to just create the Solr core from them then the url will look something like
+[source,bash]
+----
+admin/cores?action=CREATE&name=_core-name_&instanceDir=_path/to/dir_&config=solrconfig.xml&dataDir=data
+----
+====
+
+[example.tab-pane#v2coreadmin-create]
+====
+[.tab-label]*V2 API*
+
+Assumes SOLR-15372 is committed.
+
+[source,bash]
+----
+curl -X POST http://localhost:8983/api/cores -H 'Content-Type: application/json' -d'
```

Comment:
[0] I think you're missing a space between `-d` and the single-quote that starts the request-body, which I think would fail the command?

Reply:
it seems to work!   We have 32 instances of `-d'` and 152 of `-d '` so I will put the space in...